swift-blackjack-viewsTests/DeckSpec.swift

```diff
@@ -0,0 +1,48 @@
+//  DeckSpec.swift
+
+import Quick
+import Nimble
+@testable import swift_blackjack_views
+
+class DeckSpec: QuickSpec {
+    override func spec() {
+        describe("Deck") {
+            
+            var deck : Deck = Deck()
+            
+            beforeEach({ () -> () in
+                deck = Deck()
+            })
+            
+            describe("initializer") {
+                it("should set up the deck with 52 undealt cards") {
+                    expect(deck.description).to(contain("Cards Remaining: 52"))
+                    expect(deck.description).to(contain("Cards Dealt: 0"))
+                }
+            }
+            
+            describe("drawCard") {
+                it("should return the King of Clubs from a fresh deck") {
```

Comment:
yeah, it's kind of in service of checking the shuffle method, that there's an established order to start with that should then change. it should probably go into the instructions to pay attention to the order of the suits.